### PR TITLE
chore!: drop support for Node 12

### PR DESCRIPTION
**Pull request by Jawayria**
BREAKING CHANGE: Node 12 support dropped

### CODE REVIEW

File: audit-ci.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
         1067279,
         1067696,
         1069976,
-        1069985
+        1069985,
```
Comment (UsamaSadiq) [inline audit-ci.json:20]: Why is this change needed?

**Comment by Jawayria** [comment]
Same vulnerabilities on master branch as well

**Comment by jmbowman** [comment]
Any updates on this?  The FED-BOM squad says some of their work in this repo is blocked until Node 12 support is dropped.

**Comment by Jawayria** [comment]
> Any updates on this? The FED-BOM squad says some of their work in this repo is blocked until Node 12 support is dropped.

Waiting for vulnerabilities on Master branch to be fixed.

**Comment by Jawayria** [comment]
@UsamaSadiq will carry this forward

**Comment by iamsobanjaved** [comment]
Change is already merged in this commit https://github.com/openedx/frontend-app-payment/commit/51c5c81af26fb82aed4a437de66c9e4da42a8751
